src/faq.md

```diff
@@ -84,4 +88,18 @@ Next, confirm that your request includes the necessary headers:
 - `Accept-Encoding: identity` to avoid compressed responses.
 - `Connection: close` to ensure the server closes the connection after the response.
 
-If the issue persists, [enable extra logging](#faq10) with `RUST_LOG=debug` or `RUST_LOG=trace` for deeper insights into what TLSNotary is doing.
\ No newline at end of file
+If the issue persists, [enable extra logging](#faq10) with `RUST_LOG=debug` or `RUST_LOG=trace` for deeper insights into what TLSNotary is doing.
+
+If the connection failure is related to websocket and you are using the browser extension, please ensure that the domain you are connecting to is [whitelisted](https://docs.tlsnotary.org/developers/notary_server.html#websocket-proxy-server) or you will have to [run a local server](https://docs.tlsnotary.org/quick_start/browser_extension.html#websocket-proxy).
+
+### What is the difference between TLSNotary and an Oracle? { #faq12 }
+TLSNotary is designed to cryptographically prove the authenticity of HTTPS communications without revealing all data, and is not inherently connected to any blockchain systems. It can be used to verify past and/or private communications.
+
+Oracles are designed to bring off-chain information to specific blockchains. TLSNotary can be used in conjunction with oracles, but not in place of them.
```

Comment:
Thanks for bringing this up. We indeed need to explain this here. However, we need to explain it better than your attempt here. I'll file this as a separate issue